main_test.go

```diff
@@ -38,8 +38,10 @@ import (
 )
 
 const (
-	testDbDSN  = "test.db"
-	testDbType = "sqlite"
+	testDbDSN         = "test.db"
+	testDbType        = "sqlite"
+	cadenceTxBasePath = "./cadence/transactions"
+	numProposers      = 3
```

Comment:
```suggestion
	numProposalKeys      = 3
```

Reply:
Removed, this was unused as key count is now configured via `.env.test`.